cdp_scrapers/instances/portland.py

```diff
@@ -179,19 +179,24 @@ def separate_name_from_title(title_and_name: str) -> str:
     Returns
     -------
     name: str
-        tile_name_name with first word removed e.g. Ted Wheeler
+        tile_name_name with all title-related words removed e.g. Ted Wheeler
 
-    Notes
-    -----
-    first word in title_and_name is presumed to be title
     """
     # title_and_name:
     #   The title (Mayor of Commissioner) and name of a Portland City Commission
     #   member.
-    #   e.g., Mayor Ted Wheeler, Commissioner Carmen Rubio
-
-    name_index = title_and_name.find(" ")
-    return title_and_name[name_index + 1 :]
+    #   e.g., Mayor Ted Wheeler, Commissioner Carmen Rubio,
+    #   Former Commissioner Commissioner Jo Ann Hardesty
+
+    while (
+        "Former" in title_and_name
+        or "Mayor" in title_and_name
+        or "Commissioner" in title_and_name
+    ):
+        name_index = title_and_name.find(" ")
+        title_and_name = title_and_name[name_index + 1 :]
+
+    return title_and_name.strip()
```

Comment:
Shak, are these changes to `portland.py` necessary to get the pipeline pass? If not, personally, I think we should exclude this, and merge via a separate PR. I prefer to keep PR scope small, if possible. :sweat_smile:

Reply:
I made these changes to `portland.py` because the tests on GitHub failed, and I needed to ensure that the scraper would pass them. See [this discussion on Slack](https://app.slack.com/client/TMPTLMNJ2/CMA9K2F6E/thread/CMA9K2F6E-1671340475.739399) for more details.